src/web/routes/application/children-dob/handle-remove-action.js

```diff
@@ -0,0 +1,28 @@
+const { removeChildByIndex } = require('./remove-child-by-index')
+const { PATH } = require('./constants')
+
+const removeActionRequested = body => body.hasOwnProperty('remove')
+
+const decrementInputCount = childCount => childCount === 1 ? 1 : childCount - 1
+
+const handleRemoveAction = (req, res, next) => {
+  if (!removeActionRequested(req.body)) {
+    return next()
+  }
+
+  const { childCount } = req.session.children
+  const index = req.body.remove
```

Comment:
Could we update `remove` to something like `indexToRemove` or something similar. `remove` on it's own sounds like a flag to me.

Reply:
Possibly, I guess it makes sense in the context of the middleware (although I think the variable name `index` makes it obvious what the purpose is), but I think it makes the HTML more confusing:
```
<button value="1" name="indexToRemove" type="submit" id="remove-child-1">Remove</button>
```
Does this mean the add button would also need updating?
```
<input type="submit" value="Add another child" name="add" id="add-another-child">
```

Reply:
Maybe:
```
<name="removeChildIndex" value="1" />
<name="addChild" value="Add another child" />
```
?

Reply:
I kind of agree with Dave, but I'm tempted to leave it as it is because by making the JS better, we make the HTML worse, so its a flip of a coin. So I'd go with leaving it as it requires no further change.

Reply:
I've updated to use `removeChildIndex` - I've also updated the HTML to use `button` instead of `input` to be consistent with GOV.UK button component.